src/IssueViz/IssueVisualizationControl/IssueVisualizationToolWindow.cs

```diff
@@ -65,18 +65,5 @@ public IssueVisualizationToolWindow(IServiceProvider serviceProvider) : base(nul
 
             Content = new IssueVisualizationControl(viewModel);
         }
-
-        protected override void Dispose(bool disposing)
```

Comment:
We know that this doesn't work (#1714) and the tests are causing the style branch to fail unnecessarily.